src/main/resources/application.yml

```diff
@@ -180,18 +180,18 @@ kafka:
       data-type: org.folio.innreach.domain.dto.folio.inventorystorage.InstanceIterationEvent
   backoff:
     interval: ${DEFAULT_INTERVAL:95000}
-    max_failure: ${MAX_FAILURE:360}
+    max_failure: ${MAX_FAILURE:1}
```

Comment:
Do we need to have an application parameter for setting a limit on the retries during suspension? Is that what is here? We may want to retry indefinitely now that we have the outbox. Seems like there is`retry_attempts` below. 

Or is it that we're planning to get rid of `backoff` entirely with future changes to remove the old code?

Reply:
This is modified wrongly. Now putting the old value itself. As of now, even though  this param is being used in kafka retry config, this wont make any impact as the list of retryable exception will not occur at the time of persisting kafka events into DB